From bf3e7818c1114109dabd5d1c4f21f9520f735667 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Fri, 28 Feb 2025 10:42:17 +0000
Subject: [PATCH] dts: bcm2712-ds: Remove bcm7445 from gio_aon

The presence of bcm7445 in the gio_aon compatible strings should be
cosmetic, but it doesn't match gio and it breaks the pinctrl utility.
Standardise on just brcm,bcmstb-gpio, while also making a note to
improve pinctrl.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm64/boot/dts/broadcom/bcm2712-ds.dtsi | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/boot/dts/broadcom/bcm2712-ds.dtsi
+++ b/arch/arm64/boot/dts/broadcom/bcm2712-ds.dtsi
@@ -627,6 +627,7 @@
 };
 
 &gio_aon {
+	compatible = "brcm,brcmstb-gpio";
 	brcm,gpio-direct;
 };
 
